zebra-chain/src/work/difficulty.rs

```diff
@@ -320,3 +323,47 @@ impl AddAssign for Work {
         *self = *self + rhs;
     }
 }
+
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq, PartialOrd, Ord)]
+/// Partial work used to track relative work in non-finalized chains
+pub struct PartialCumulativeWork(usize);
```

Comment:
`usize` is 32-bit on some platforms. But like `Work`, `PartialCumulativeWork` needs to be `u128`.

The current Zcash mainnet does approximately 2^33 `Work` per second, and each block is approximately 75 seconds worth of `Work`. So each block has approximately 2^39 `Work`.

And we want to be able to scale to a much larger Zcash network in future. See this comment for more details:
https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/work/difficulty.rs#L107

```suggestion
pub struct PartialCumulativeWork(u128);
```